firstvoices/backend/tests/test_apis/test_sites_api.py

```diff
@@ -592,14 +592,14 @@ def test_detail_team_access(self):
         assert response_data["slug"] == str(site.slug)
 
     @pytest.mark.django_db
-    def test_detail_403(self):
+    def test_detail_401(self):
         site = factories.SiteFactory.create(visibility=Visibility.MEMBERS)
         user = factories.get_non_member_user()
         self.client.force_authenticate(user=user)
 
         response = self.client.get(f"{self.get_detail_endpoint(site.slug)}")
 
-        assert response.status_code == 403
+        assert response.status_code == 401
```

Comment:
Ohhhh sorry, this should only return 401 if the user is not logged in. If they lack permissions, 403 is the correct response.

Reply:
Ah apologies I should have asked for clarification on the ticket. I'll adjust the PR to only return a 401 if the user is unauthenticated!